cfdi.go

```diff
@@ -100,7 +99,7 @@ func NewDocument(env *gobl.Envelope) (*Document, error) {
 
 		CFDIRelacionados: newCfdiRelacionados(inv),
 		Emisor:           newEmisor(inv.Supplier),
-		Receptor:         newReceptor(inv.Customer, lookupUsoCFDI(inv)),
+		Receptor:         newReceptor(inv.Customer),
```

Comment:
A (welcome) consequence of the change discussed [here](https://github.com/invopop/gobl/pull/183#issuecomment-1683465103) (and that weakens my arguments against it): one less parameter to pass now that GOBL includes the attribute in the same entity as the CFDI.